app/components/EditStudent.js

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+
+export default function EditStudent(props) {
+  console.log('EditStudent - props: ', props);
+  const {
+    firstName, lastName, email, campus, campuses,
+    handleFirstNameChange, handleLastNameChange, handleEmailChange, handleCampusChange, handleSubmit
+  } = props;
+
+  return (
+    <div className="col-xs-12">
+      <h2>EDIT STUDENT</h2>
+      <form className="form-horizontal" onSubmit={(e) => handleSubmit(e)}>
+        <div className="form-group">
+          <label className="col-xs-12">First Name:</label>
+          <input type="text" className="form-control" id="firstName"
+            onChange={(e) => handleFirstNameChange(e)} value={firstName}></input>
```

Comment:
In this particular case, we don't need to write our own function to call the `handleFirstNameChange` with the event.

This:
```
onChange={(e) => handleFirstNameChange(e)}
```

Would be equivalent to this: 
```
onChange={handleFirstNameChange}
```